bitnami/kube-prometheus/values.yaml

```diff
@@ -76,13 +76,22 @@ operator:
   ##
   # schedulerName:
 
-  ## SecurityContext configuration
+  ## Prometheus Operator pods' Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
   ##
-  securityContext:
+  podSecurityContext:
     enabled: true
     runAsUser: 1001
     fsGroup: 1001
 
+  ## Prometheus Operator containers' Security Context (only main container)
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ##
+  containerSecurityContext:
+    enabled: true
+    allowPrivilegeEscalation: false
+    readOnlyRootFilesystem: false
+
```

Comment:
This one could be more strict.

Reply:
I think that we can start like this (using those security policies).